pkg/profileparser/profileparser.go

```diff
@@ -328,7 +328,7 @@ func parseProfileFromNode(profileRoot *xmlquery.Node, pb *cmpv1alpha1.ProfileBun
 		// In case the profile sets its own CPE string
 		productType, productName := getProductTypeAndName(profileObj, defType, defName)
 
-		if productType == "Platform" && utils.GetPlatform() == "ROSA" {
+		if strings.EqualFold(string(productType), "Platform") && strings.EqualFold(utils.GetPlatform(), "ROSA") && utils.IsHostedControlPlane() {
```

Comment:
Nit: I'm okay with this, but I wonder if we could / should just condition this to an external control plane, instead of external control plane and ROSA platfom.

Reply:
We can't do that, because we support platform profiles on HyperShift